examples/pythonDriver/sedovWithStressFunction/run_sedov_problem.py

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
Here's an example where I run a problem derived from the sedov example.  Before running, I set the values of s_xx, s_yy, and s_zz equal to sin(2*pi*r / max(r)) using pygeosx.  Here's a snapshot of the model at t=0s in visit:

![image](https://user-images.githubusercontent.com/4932756/92980985-e8e16980-f44c-11ea-85bf-9a61b4b28ab6.png)